src/t8_cmesh/t8_cmesh_readmshfile.cxx

```diff
@@ -1710,6 +1767,18 @@ t8_cmesh_from_msh_file (const char *fileprefix, int partition,
    * not know its dimension on these processes. */
   t8_cmesh_set_dimension (cmesh, dim);
 
+  /* Register the geometries for the cmesh. */
+  const int           registered_geom_success =
+    t8_cmesh_from_msh_file_register_geometries (cmesh, use_occ_geometry, dim,
+                                                fileprefix, &linear_geometry,
+                                                &occ_geometry);
+  if (!registered_geom_success) {
+    /* Registering failed */
+    t8_debugf ("OCC is not linked. Cannot use OCC geometry.\n");
```

Comment:
This should be a `t8_errorf()`. When compiling the code in release, the user has no idea why his cmesh will be destroyed.